tests/scripts/kubeadm.sh

```diff
@@ -41,7 +41,7 @@ EOF
     echo "wait for K8s master node to be Ready"
     INC=0
     while [[ $INC -lt 20 ]]; do
-        kube_ready=$(kubectl get node -o jsonpath='{.items[0].status.conditions[3].status}')
+        kube_ready=$(kubectl get node -o jsonpath='{.items['$count'].status.conditions[?(@.reason == "KubeletReady")].status}')
```

Comment:
this is some fancy looking bash here.  is the conditions index number not always the same i take it?

Reply:
Previously we relied on index 3, but now it's index 4. Correct, this gets us away from relying on the index.